config/core/resources/eventtype.yaml

```diff
@@ -142,108 +158,6 @@ spec:
     name: v1beta2
     served: true
     storage: false
-    schema:
```

Comment:
Just curious as I'm still learning how all of the API stuff works, why are we removing the `schema` from `v1beta2` here?

Reply:
because schema is now same, allow the conversion to also have `.reference` on the old type.